src/cocotb_tools/makefiles/Makefile.inc

```diff
@@ -47,7 +47,7 @@ COCOTB_PY_DIR := $(shell cocotb-config --prefix)
 
 # Directory containing all support files required to build cocotb-based
 # simulations: Makefile fragments, and the simulator libraries.
-COCOTB_SHARE_DIR := $(COCOTB_PY_DIR)/cocotb/share
+COCOTB_SHARE_DIR := $(shell cocotb-config --share)
```

Comment:
This doesn't need to be done in this commit. OR take the opportunity to deprecate a remove some variables that are equivalent to cocotb-config calls.